### ISSUE TITLE
package not recognizing source language correctly

### ISSUE THREAD

**Issue by Octofoxy** [sxtfv/flutter_arb_translator#1]
I'm trying to translate a german .arb file into an english one. However, the package assumed the source language to be english, tried to translate it back into english, and in the end did no translation at all.

Here's a (shortened) version of the file app_de.arb:
`{
  "questions": "Fragen",
  "account": "Konto"
}`
  
I'm using this command:
`flutter pub run flutter_arb_translator:main --from de --to en --service deepl`

Output:

02-03-2023 12:40:25 [ARBTranslator] TRC: Started translation from en to [en]
02-03-2023 12:40:25 [ARBTranslator] TRC: Total entries count: 2
02-03-2023 12:40:25 [ARBTranslator] TRC: Unmodified entries count: 0
02-03-2023 12:40:25 [ARBTranslator] TRC: To translate entries count: 2

==============================================================
questions
[en] [SRC]: "Fragen"
[en] [ADD]: "Fragen"

==============================================================
account
[en] [SRC]: "Konto"
[en] [ADD]: "Konto"

I tried adding `"@@locale" : "de",` to the top of the .arb file, but it had no effect.

**Comment by sxtfv** [comment]
Hello, sorry for missing this. I will check and update in 1-2 days

**Comment by sxtfv** [comment]
@Octofoxy I just published update (1.0.5) with a fix for this. Please check

**Comment by fly-qp** [comment]
Same issue happening with deepl & latest version as well i tried downgrading to 1.0.6 (not working either)

```
flutter pub run flutter_arb_translator:main --from en --to pt --service deepl -k failureDbUnknown
Deprecated. Use `dart run` instead.
05-29-2023 01:37:32 [ARBTranslator] TRC: Started translation from en to [pt]
05-29-2023 01:37:32 [ARBTranslator] TRC: Total entries count: 213
05-29-2023 01:37:32 [ARBTranslator] TRC: Unmodified entries count: 0
05-29-2023 01:37:32 [ARBTranslator] TRC: To translate entries count: 1
==============================================================
failureDbUnknown
[en] [SRC]: "An unknown error occurred. Please try again later or contact support for further assistance."
[pt] [ADD]: "An unknown error occurred. Please try again later or contact support for further assistance."
```

But once this package works it will be really huge  <3
